src/include/distributed/pg_dist_partition.h

```diff
@@ -54,6 +54,10 @@ typedef FormData_pg_dist_partition *Form_pg_dist_partition;
 #define DISTRIBUTE_BY_RANGE 'r'
 #define DISTRIBUTE_BY_NONE 'n'
 #define REDISTRIBUTE_BY_HASH 'x'
+#define COORDINATOR_TABLE 'l'
```

Comment:
I find `IsNoDistributionKeyMethod ` or in general  all the names with `NoDistributionKeyMethod` a little  unusual. 

What about `CitusTableWithoutDistributionKey`?

Reply:
I think macro name should be `IsNoDistributionKeyMethod` as we're checking for a distribution method but I should update the comments "no-distribution-key" to "citus table without dist key" or something like this. Does it sound good ?

Reply:
To me `NoDistributionKeyMethod` is very confusing. I don't think we have anything called `DistributionKeyMethod` in the code-base, we have `DistributionMethod`, and I  get that `NoDistributionKeyMethod ` has some meaning associated with it, but it confuses me.

`DistributionMethodWithoutKey`  could be another  alternative. I still vote for `CitusTableWithoutDistributionKey `